frame/system/benchmarking/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "frame-system-benchmarking"
+version = "2.0.0-dev"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+license = "GPL-3.0"
+homepage = "https://substrate.dev"
```

Comment:
I'm guessing we want to link to substrate.io now?

Reply:
sounds like a bigger change for all of substrate. maybe another pr